parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -27,13 +27,16 @@
 from parsl.executors.high_throughput.probe import probe_addresses
 from parsl.multiprocessing import ForkProcess as mpForkProcess
 from parsl.multiprocessing import SpawnProcess as mpSpawnProcess
-
+from parsl.log_utils import set_file_logger
 from parsl.multiprocessing import SizedQueue as mpQueue
 
 from parsl.serialize import unpack_apply_message, serialize
 
 HEARTBEAT_CODE = (2 ** 32) - 1
 
+# Note __name__ could be "__main__".
+logger = logging.getLogger("parsl.executors.high_throughput.process_worker_pool")
```

Comment:
Previously this code generated several distinct log files: a `manager.log` which contained log information from the manager process; and per-worker logs, worker_{}.log for each of the worker processes.

Because of the way the process_worker_pool code is structured, the workers inherit (a forked copy of) any handlers configured in the manager process - to direct `parsl` logs to the manager log file.

On top of this, the workers then initialise their own handler to direct `worker_log` logs to the individual worker log files.

Because they use a different logger name, log lines aimed at `worker_log` are only handled by the worker log file handler, not by the manager log file handler - but that's only because they use a different name, not because they are in different processes.

Moving all of the workers and the manager to use a single name in this line of code breaks that separation: run pytest with htex and observe that `grep "All processing finished for executor tas" manager.log ` shows worker log lines appearing in the manager log with this PR applied, but not otherwise.

I'm not sure what the right thing to do here is - maybe at the point right after forking the worker process, drop existing handlers and start with a fresh clean logging environment, rather than inheriting it?